override_mail_recipients/models/ir_mail_server.py

```diff
@@ -1,45 +1,34 @@
 """Make sure no unwanted mail leaves the server."""
 # Copyright 2015-2019 Therp BV <https://therp.nl>.
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
-import re
 
 from email.utils import COMMASPACE
 
 from odoo import api, models
-from odoo.addons.base.ir.ir_mail_server import extract_rfc2822_addresses
+from odoo.addons.base.models.ir_mail_server import extract_rfc2822_addresses
 
 
 ADDRESS_REPLACEMENTS = {
-    '\\': '',  # Remove backslash
-    '"': '\\"',  # escape double quote
-    '<': '[',  # open actual address part
-    '>': ']',  # close actual address part
-    '@': '(at)'}
-REPLACE_KEYS = sorted(ADDRESS_REPLACEMENTS, key=len, reverse=True)
-REPLACE_REGEX = re.compile('|'.join(map(re.escape, REPLACE_KEYS)))
+    ord("\\"): "",  # Remove backslash
```

Comment:
Why the ord()?

Reply:
That's how `str.translate` works: https://docs.python.org/3/library/stdtypes.html#str.translate

It's a little odd, but I think it's because it only replaces single characters. I'll use `str.maketrans` to make it clearer.